src/main/resources/schema/preservation.graphqls

```diff
@@ -30,6 +30,7 @@ enum EdgeType {
     IdentityDiscontinuableStatement
     IdentityWelcomeBehaviour
     IdentityIntentDetectionSkill
+    IdentityDisambiguationBehaviour
```

Comment:
we should always append to the end of the enum instead of in the middle to avoid weird stuff because enums are sometimes rerefenced via their ordinal